public/js/components/Sources.js

```diff
@@ -20,7 +20,7 @@ const Sources = React.createClass({
 
   render() {
     const { sources, selectSource } = this.props;
-    const modifierTxt = Services.appinfo.OS === "Darwin" ? "Cmd" : "Ctrl";
+    const modifierTxt = Services.appinfo.OS === "Darwin" ? "⌘" : "Ctrl";
```

Comment:
nice - it would be nice if we had a util method `cmdString` so that we don't duplicate this everywhere.

maybe inside of utils? We can change this when we add l10n